compiler/crates/graphql-syntax/src/lib.rs

```diff
@@ -81,17 +92,6 @@ pub fn parse_executable_with_error_recovery_and_parser_features(
     parser.parse_executable_document()
 }
 
-/// Parses a GraphQL document that's restricted to executable
-/// definitions with custom feature flags passed as `features`.
-pub fn parse_executable_with_features(
-    source: &str,
-    source_location: SourceLocationKey,
-    features: ParserFeatures,
-) -> DiagnosticsResult<ExecutableDocument> {
-    parse_executable_with_error_recovery_and_parser_features(source, source_location, features)
-        .into()
-}
```

Comment:
Is this a meaningful change, or is the function just moving?

Reply:
Just moving the function around. The grouping felt weird.